snapshots/devmapper/losetup/losetup.go

```diff
@@ -78,6 +79,10 @@ func losetup(args ...string) (string, error) {
 	data, err := exec.Command("losetup", args...).CombinedOutput()
 	output := string(data)
 	if err != nil {
+		if strings.Contains(output, "No such file or directory") || strings.Contains(output, "No such device") {
```

Comment:
hopefully this string comparison is enough across distros!

Reply:
needs LANG=C ?

Reply:
@AkihiroSuda good point, better to ensure we will get the right language.

Reply:
@AkihiroSuda done, PTAL.